account_financial_risk/models/res_config.py

```diff
@@ -13,3 +13,8 @@ class AccountConfigSettings(models.TransientModel):
     allow_overrisk_invoice_validation = fields.Boolean(
         related="company_id.allow_overrisk_invoice_validation", readonly=False
     )
+    portal_show_financial_risk = fields.Boolean(
```

Comment:
It would be interesting also to have the option by partner (apart from this one), enabled by default, but invisible if this parameter is not set.